src/ts/component/block/embed.tsx

```diff
@@ -644,6 +644,10 @@ const BlockEmbed = observer(class BlockEmbed extends React.Component<I.BlockComp
 					// Fix Bilibili schemeless urls
 					if (block.isEmbedBilibili() && text.match(/src="\/\/player[^"]+"/)) {
```

Comment:
but it will change autoplay only if the link is incorrect this way

Reply:
The URL copied from the website simply lacks https:, which generally is not a problem. But it's still worth considering this situation.